<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BulkLabelStatistics</class>
 <widget class="qMRMLWidget" name="BulkLabelStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>758</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="segmentationsDirPushButton">
     <property name="text">
      <string>Segmentations Directory...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="segmentationsDirLabel">
     <property name="text">
      <string>No labelmaps directory specified</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="volumesDirPushButton">
     <property name="text">
      <string>Volumes Directory...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="volumesDirLabel">
     <property name="text">
      <string>No volumes directory specified</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>158</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="outputFilePushButton">
     <property name="text">
      <string>Output File ...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="outputFileLabel">
     <property name="text">
      <string>No output file specified</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="computeStatisticsPushButton">
     <property name="text">
      <string>Compute Segment Statistics</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
